15/umbraco-deploy/upgrades/version-specific.md

```diff
@@ -4,108 +4,50 @@ description: >-
-- `IArtifactImportExportService.ExportArtifactsAsync(...)`: the artifacts parameter is updated to `IAsyncEnumerable<IArtifact>`, so artifacts are asynchronously created when iterating;
-- `IServiceConnector.GetArtifactsAsync(...)` and `IServiceConnectorFactory.GetArtifactsAsync(...)`: these new extension methods call `GetArtifactAsync(...)` on the relevant service connectors and returns `IAsyncEnumerable<IArtifact>`.
 
-#### Signature notification handling
-Deploy stores artifact signatures (hashes) in the database to avoid creating the artifact to for example compare them between disk/database or environments. To ensure the signatures are always up-to-date, the `ISignatureService` used custom notification handling via the `RegisterHandler(...)` and `HandleExternalNotification(...)` methods (because Deploy suppresses notifications during deployment). CMS version 12 introduced the `IDistributedCacheNotificationHandler` marker interface and Deploy 12 the `IDeployRefresherNotificationHandler` that will still handle the notifications during deployments. The following methods are therefore removed:
-- `RegisterHandler(...)`: register your handler using the standard `builder.AddNotificationHandler<>()` or `builder.AddNotificationAsyncHandler()` method;
-- `HandleExternalNotification(...)`: implement your handler using `IDeployRefresherNotificationHandler<TNotification>` or `IDeployRefresherNotificationAsyncHandler<TNotification>`.
+Version 15 of Umbraco Deploy has a minimum dependency on Umbraco CMS core of `15.0.0`. It runs on .NET 9.
 
-#### Return type of Data Type configuration has changed to a dictionary
-The return type of `IDataTypeConfigurationConnector.FromArtifact()/FromArtifactAsync()` has changed from `object` to `IDictionary<string, object>` to align with the Data Type configuration changes in the CMS (see [PR #13605](https://github.com/umbraco/Umbraco-CMS/pull/13605)). Also, obsoleted methods and default interface implementations on Deploy interfaces have been removed (see [PR #15965](https://github.com/umbraco/Umbraco-CMS/pull/15965)).
+### Breaking changes
 
-#### Removed code due to changes in the CMS
-The CMS removed support for the legacy Media Picker, Grid layout, Nested Content editors and macros, which means Deploy doesn't provide support for transferring these editors. The following related code has been removed:
-- `IGridCellValueConnector`, `IGridCellValueConnector2` and `IGridCellValueConnectorFactory`: including all Grid cell value connector implementations;
-- `GridDataTypeConfigurationConnector`, `NestedContentDataTypeConfigurationConnector`;
-- `IMacroParser` (in the CMS), `MacroParserBase` and `MacroParser`;
-- `MediaPickerValueConnector` and `MacroConnector`.
+Version 15 contains breaking changes. Not many projects are expected to be affected by them, as they are in relevant areas when extending Deploy to support additional entities and/or property editors. For reference though, the full details are listed here:
```

Comment:
Suggestion to shorten a sentence, and avoid mentioning anything about how many will be affected - I don't think this is relevant 🙌

```suggestion
Version 15 contains breaking changes. The breaking changes appear in areas related to extending Deploy to support additional entities and property editors. For reference though, the full details are listed here:
```